### ISSUE TITLE
Downloading coincidence fails with to many stations

### ISSUE THREAD

**Issue by 153957** [HiSPARC/publicdb#66]
As mentioned in https://github.com/HiSPARC/sapphire/issues/30, when attempting to download coincidences for a cluster which has more than 32 stations the download will fail.
Perhaps we should change the download to work with subclusters instead of clusters.. but that is only a partial workaround..

Example:
```
Parameters
Cluster: Amsterdam
N: 6
Start: 2013/10/02
End: 2013/10/03
```
http://data.hisparc.nl/data/network/coincidences/?download=False&cluster=Amsterdam&n=6&end=2013-10-03+00%3A00%3A00&start=2013-10-02+00%3A00%3A00

**Comment by 153957** [comment]
Partly fixed (9781965e087a208e5ffbf0d4442ed4a40dd38079) by throwing error when a cluster is selected with to many stations. Not ideal, but at least tells the user why there are no coincidences returned..

**Comment by 153957** [comment]
Untill this is fixed in SAPPHiRE/PyTables/NumPy I believe it is handled here correctly.
Once it is fixed and more arguments are allowed we can easily remove the checks..
